ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -213,6 +216,23 @@ public CloudBlobMetadata setContainerId(int containerId) {
     return this;
   }
 
+  /**
+   * @return whether the blob was cloud encrypted.
+   */
+  public boolean getUsesCloudEncryption() {
+    return usesCloudEncryption;
+  }
+
+  /**
+   * Set the encryption context.
```

Comment:
Minor: "context" seems weird here since it's not the name of method or argument.  Maybe: set the cloud encryption flag?

Reply:
Yeah I had originally had it as the encryption context string but changed it later.  Changed the comment